domains/sctesting2023.is-cool.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-cool.dev",
+    "subdomain": "sctesting2023",
+
+    "owner": {
+        "email": "goryo.webdev@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "sctesting2023.is-cool.dev.hosting.netsuite.com"
```

Comment:
404 CNAME.

Reply:
I was folowing the instructions from https://www.anchorgroup.tech/how-to-set-up-suitecommerce-domain. 

What step you recemmend?

Reply:
If you've followed those, I can go ahead and approve.